README.md

```diff
@@ -123,7 +140,7 @@ let vgg19 () =
   in
   input_id, model
 ```
-This model is used in the [following example](https://github.com/LaurentMazare/tensorflow-ocaml/blob/master/examples/fnn/vgg19.ml) to classify any input image, in order to use it you will have to download some [pre-trained weights](https://github.com/LaurentMazare/tensorflow-ocaml/releases/download/0.0.7/vgg19.cpkt).
+This model can be used to classify an input image. In order to use it you will have to download the [pre-trained weights](https://github.com/LaurentMazare/tensorflow-ocaml/releases/download/0.0.7/vgg19.cpkt).
```

Comment:
Could we put back the (correct) link for the example ? It should be https://github.com/LaurentMazare/tensorflow-ocaml/blob/master/examples/neural-style/vgg19.ml .